src/AppProviders.tsx

```diff
@@ -68,15 +68,18 @@ export function AppProviders({ children }: ProvidersProps) {
                         <WalletProvider>
                           <KeepKeyProvider>
                             <WalletConnectV2Provider>
-                              <ModalProvider>
-                                <ErrorBoundary FallbackComponent={ErrorPage} onError={handleError}>
-                                  <ActionCenterProvider>
+                              <ActionCenterProvider>
+                                <ModalProvider>
+                                  <ErrorBoundary
+                                    FallbackComponent={ErrorPage}
+                                    onError={handleError}
+                                  >
                                     <AppProvider>
                                       <DefiManagerProvider>{children}</DefiManagerProvider>
                                     </AppProvider>
-                                  </ActionCenterProvider>
-                                </ErrorBoundary>
-                              </ModalProvider>
+                                  </ErrorBoundary>
+                                </ModalProvider>
+                              </ActionCenterProvider>
```

Comment:
The story of our life, changing providers orders all days 😆 

Shouldn't the error boundary be in the upper scope possible? 🤔

Reply:
Error-boundary fix should be its own PR though, definitely weird shenanigans happening re: suspense